v_3_2_5/master_template.go

```diff
@@ -2298,7 +2298,7 @@ coreos:
       RestartSec=0
       TimeoutStopSec=10
       EnvironmentFile=/etc/network-environment
-      Environment="IMAGE=quay.io/giantswarm/hyperkube:v1.9.5"
+      Environment="IMAGE=quay.io/giantswarm/hyperkube:v1.10.0"
```

Comment:
We could template this version actually and hard code it in the versioned directories here.

Reply:
:+1:  +1